cmd/snap/cmd_run_test.go

```diff
@@ -75,6 +76,15 @@ hooks:
  configure:
 `)
 
+func getExcludedSyscalls() string {
```

Comment:
Instead of duplicating the code, I'd probably export the method from the `strace` package and invoke it directly. Another option (and actually my preferred one, since we are testing the `cmd_run.go` file and not the `strace` package — but I suspect that my colleagues might disagree) is to not even check the whole command line, but only those parts that directly depend on the code in `cmd_run.go`; in other words, I would remove at least the `-e` option from the parameters array before comparing it; or just compare those parts of the command-line that we care about (if any).

Reply:
I feel like i should just export the variable which has computed the values and be done with it. If we were already doing it in one test package, and had copies of the value in the other package too, it is time to just export it once and use it everywhere.